### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by Kandy-Peter**
In this PR, I added the following elements:

1. Downloaded the [Microverse  template](https://github.com/microverseinc/curriculum-template-databases)
2. Created a database named `vet_clinic`.
3. Created a table named `animals` with the following columns:
      `id`: integer
      `name`: string
      `date_of_birth`: date
      `escape_attempts`: integer
      `neutered`: boolean
     `weight_kg`: decimal
4. copied the created table into the `schema.sql` file in the generated repository.
5. Insert some data into the `data.sql` file in the generated repository.
6. Write queries for some requirements and add them to the `queries.sql` file

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true,  10.23);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true,  8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, false,  15.04);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true,  11);
```
Comment (andersonlebon) [inline data.sql:6]:
```sql
CREATE TABLE animals (
    name varchar(100)
    id INT NOT NULL PRIMARY KEY,
    name VARCHAR(100) NOT NULL,
    date_of_birth DATE NOT NULL,
    escape_attempts INT NOT NULL,
    neutered BOOLEAN NOT NULL,
    weight_kg DECIMAL(10, 2) NOT NULL
);
```
- [ ] Kindly note according to the code you wrote above :point_up: is not the id should not be null when inserting data in the table, see below :point_down: the error I am getting

image


- [x] You can fix this either by auto-generating your id in the animals' table or by adding the id manually when inserting data in the table. 

### by auto-generating the id
```sql
CREATE TABLE animals (                                                                                                      
    id INT GENERATED ALWAYS AS IDENTITY NOT NULL,
    name VARCHAR(100) NOT NULL,
    date_of_birth DATE NOT NULL,
    escape_attempts INT NOT NULL,
    neutered BOOLEAN NOT NULL,
    weight_kg DECIMAL(10, 2) NOT NULL
);
```

##$ by inserting the id manually 
```sql
INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true,  10.23);
``

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @Kandy-Peter 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Kandy-Peter) [reply data.sql:6]: Thank you @andersonlebon  for the feedback!

**Review by andersonlebon — APPROVED** [review]
## Hi @Kandy-Peter 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**